rewrite-java/src/main/java/org/openrewrite/java/cleanup/IsBlockLambdaConvertibleVisitor.java

```diff
@@ -0,0 +1,115 @@
+package org.openrewrite.java.cleanup;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
We would actually like to get rid of the Apache Commons dependencies. So I suggest you instead introduce a custom Lombok `@Value` class for this purpose.